### ISSUE TITLE
[Feature Request]: `Microsoft.Synapse/workspaces` module is missing options to deploy the `bigDataPools`

### ISSUE THREAD

**Issue by Lddeiva** [Azure/bicep-registry-modules#2371]
### Description

Reference: - https://learn.microsoft.com/en-us/azure/templates/microsoft.synapse/workspaces/bigdatapools?pivots=deployment-language-bicep

**Comment by AlexanderSehr** [comment]
Migrated this issue from CARML. Note, the module is currently orphaned (ref: https://azure.github.io/Azure-Verified-Modules/indexes/bicep/bicep-resource-modules/)

**Comment by sajwvanzundert** [comment]
It's been almost 2 years since this issue is opened and nothing has changed. Is anyone on the AVM team going to take responsibility for this and take ownership so the Synapse module can be finished?

**Comment by hundredacres** [comment]
@sajwvanzundert the team is always looking for folks to become module owners. Are you willing to volunteer?

**Comment by sajwvanzundert** [comment]
@hundredacres No, I don't have that time and I simply don't want to. Also, only Microsoft FTE's can be owners as per [documentation](https://azure.github.io/Azure-Verified-Modules/faq/#can-i-be-an-avm-module-owner-if-im-not-a-microsoft-fte).

**Comment by sajwvanzundert** [comment]
> [@sajwvanzundert](https://github.com/sajwvanzundert) the team is always looking for folks to become module owners. Are you willing to volunteer?

No, I don't have that time and I simply don't want to. Also, only Microsoft FTE's can be owners as per [documentation](https://azure.github.io/Azure-Verified-Modules/faq/#can-i-be-an-avm-module-owner-if-im-not-a-microsoft-fte).

**Comment by matebarabas** [comment]
@TomazMlakar, can you please take a look at this one? Thanks!
